content/en/blog/2024/inpod-traffic-redirection-ambient/index.md

```diff
@@ -0,0 +1,217 @@
+network namespace, it is perfectly possible to leverage Linux’s low-level socket API to allow a process running in one
+network namespace to create listening sockets in another network namespace, assuming the target network namespace is known
+at creation time.
+- The node-local ztunnel internally spins up a new proxy instance and listen port set, dedicated to the newly-enrolled pod.
+- Once the in-Pod redirect rules are in place and the ztunnel has established the listen ports, the pod is enrolled in the
+mesh and traffic begins flowing thru the node-local ztunnel, as before.
+
+Here’s a basic diagram showing the pod enrollment flow:
+
+{{< image width="100%"
+    link="./pod-enrollment.png"
+    alt="pod enrollment flow"
+    >}}
+
+Once the enrollment is completed, traffic to and from pods in the mesh will be fully encrypted by default, as always with Istio.
+
+Traffic will now enter and leave the pod network namespace as encrypted traffic - it will look like every ambient-enrolled
+pod has the ability to enforce mesh policy and securely encrypt traffic, even though the user application running in the pod
+has no awareness of either.
+
+Here’s a diagram to illustrate how encrypted traffic flows between meshed pods in the current ambient model:
+
+{{< image width="100%"
+    link="./traffic-flows-between-mesh-pods.png"
+    alt="HBONE traffic flows between meshed pods"
+    >}}
+
+And, as before, unencrypted plaintext traffic from outside the mesh can still be handled and policy enforced, for use cases
+where that is necessary:
+
+{{< image width="100%"
+    link="./traffic-flows-plaintext.png"
+    alt="Plain text traffic flow between meshed pods"
+    >}}
+
+### Istio Ambient Traffic Redirection: What This Gets Us
+
+The end result of the new ambient capture model is that all traffic capture and redirection happens inside the pod’s network
+namespace. To the node, the CNI, and everything else, it looks like there is a sidecar proxy inside the pod, even though
+there is, as before, **no sidecar proxy running in the pod** at all. Remember that the job of CNI implementations is to get
+packets **to and from** the pod. By design and by the CNI spec, they do not care what happens to packets after that point.
+
+This approach automatically eliminates conflicts with a wide range of CNI and NetworkPolicy implementations, and drastically
+improves Istio ambient mesh compatibility with all major managed Kubernetes offerings across all major CNIs.
+
+## Wrapping Up
+
+With gracious support from the community on testing the change with various Kubernetes platforms and CNIs, and many rounds
+of reviews from Istio maintainers, we are glad the [ztunnel](https://github.com/istio/ztunnel/pull/747) and [istio-cni](https://github.com/istio/istio/pull/48253) PRs merged to Istio 1.21 so our users
+can start running ambient on any Kubernetes platforms with any CNIs in Istio 1.21 or newer. We’ve tested this with Google,
```

Comment:
nit: Google,Microsoft,Amazon OR GCP,Azure,AWS, OR GKE,AKS,EKS. We are consistent here

Reply:
I thought the same, but AWS is Amazon Web Services, Inc. rather than just Amazon when I double checked.

Reply:
updated.